app/src/main/java/com/pajato/android/gamechat/game/GameManager.java

```diff
@@ -15,11 +15,188 @@
  */
 package com.pajato.android.gamechat.game;
 
+import android.support.design.widget.Snackbar;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentActivity;
+import android.view.View;
+
+import com.pajato.android.gamechat.R;
+import com.pajato.android.gamechat.fragment.SettingsFragment;
+import com.pajato.android.gamechat.fragment.TTTFragment;
+
+import java.util.ArrayList;
+
 /**
- * Provides the interfaces that must be implemented by an game manager implementation.
+ * Manages the game related aspects of the GameChat application. These include the creation of new
+ * game instances, notifications, and game settings.
  *
- * @author Paul Michael Reilly
+ * @author Bryan Scott
  */
-public interface GameManager {
+public enum GameManager {
+    instance;
+
+    // Public Class Constants
+    public static final int SETTINGS_INDEX = 0;
+    public static final int TTT_INDEX = 1;
+    public static final int CHECKERS_INDEX = 2;
+    public static final int CHESS_INDEX = 3;
+
+    // Public Instance Variables
+    /** The Current Match History */
+    public ArrayList<String> mInstructions = new ArrayList<>();
+
+    // Private class constants
+    /** The logcat tag constant. */
+    private static final String TAG = GameManager.class.getSimpleName();
+
+    // Private instance variables
+    /** Contains the list of all the fragments. */
+    private ArrayList<Fragment> fragmentList = new ArrayList<>();
+    /** Contains the fragment index */
+    private int currentFragment;
+
+    // Public instance methods
+
+    public void init(final FragmentActivity context) {
+        currentFragment = -1;
+        mInstructions.clear();
+        fragmentList.clear();
+        //TODO: Implement these and uncomment them when they are implemented.
```

Comment:
Instead of adding all the game instances at init time, find another way to add them dynamically.  I would suggest this is an opportunity to leverage an EventBus such that an "addNewGame" event will be emitted carrying along a possibly cached instance payload of the game.  Each game would implement a clear or reset operation to support the "playAgain" event.